Added steps for installing the CLI on Apple M1 chips (#423)
astronomer/docs: cloud/stable/02_develop/01_cli-quickstart.md

curl -sSL https://install.astronomer.io | sudo bash -s < /dev/null
```

#### Install the CLI on Apple M1 Machines

> To install the Astronomer CLI on machines using the Apple M1 chip, run the following curl command:

Reviewer: ```suggestion
To install the Astronomer CLI on machines using the [Apple M1 chip](https://www.apple.com/newsroom/2020/11/apple-unleashes-m1/), run the following curl command:
```

> To install the Astronomer CLI on machines using the Apple M1 chip, run the following curl command:

Reviewer: Was confused at a glance on whether the user had to choose between the steps below or do all of them. Maybe make it slightly clearer up top that it's either (1) curl or (2) homebrew?

> To install the Astronomer CLI on machines using the [Apple M1 chip](https://www.apple.com/newsroom/2020/11/apple-unleashes-m1/), you can use either curl or Homebrew. To install using curl, run the following command:

Reviewer: ```suggestion
To install the Astronomer CLI on a machine using the [Apple M1 chip](https://www.apple.com/newsroom/2020/11/apple-unleashes-m1/), you can use either cURL or Homebrew. To install using cURL, run the following command:
```

```sh
sudo curl -sSL https://install.astronomer.io | sudo bash -s
```

> Because the Astronomer CLI does not yet have an ARM64 build, installing the Astronomer CLI via Homebrew on machines using the Apple M1 chip requires a few additional steps. If you prefer to use Homebrew for installation:

Reviewer: ```suggestion
Because the Astronomer CLI does not yet have an ARM64 build, installing the Astronomer CLI via Homebrew on machines using the Apple M1 chip requires a few additional steps. If you prefer to use Homebrew instead of cURL for installation:
```

Author: Think I resolved this with a previous edit

> Because the Astronomer CLI does not yet have an ARM64 build, installing the Astronomer CLI via Homebrew on machines using the Apple M1 chip requires a few additional steps. If you prefer to use Homebrew for installation:

Reviewer: ```suggestion
Because the Astronomer CLI does not yet have an ARM64 build, installing it via Homebrew on a machine that runs on the Apple M1 chip requires a few additional steps. If you prefer to use Homebrew for installation:
```

Author: Don't think "running" or "using" are necessarily right here, gonna keep it simple and say "a machine with an apple m1 chip"

1. Run the following command to install the x86_64 version of Homebrew:

    ```sh
    arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
    ```

> 2. Confirm that the script has successfully installed Homebrew on /usr/local.

Reviewer: ```suggestion
2. Confirm that the script was successfully installed Homebrew on `/usr/local`.
```

3. Run the following command to install the Astronomer CLI:

    ```sh

>     arch -x86_64 /usr/local/Homebrew/bin/brew install astronomer/tap/astro@0.25.1

Reviewer: This installs version `0.25.1` of the CLI, presumably. If we remove `0.25.1` would the user get latest? Let's do that if so :)

    ```

> If you still have issues during installation, ensure that Rosetta 2 is installed on your machine and try again.

Reviewer: ```suggestion
If you still have issues during installation, ensure that Rosetta 2 is installed on your machine and try again.
```

Link to Rosetta 2?
